provisioner/ibm-s3fs-provisioner.go

```diff
@@ -85,12 +92,26 @@ const (
 	autoBucketNamePrefix = "tmp-s3fs-"
 	fsType               = ""
 	caBundlePath         = "/tmp/"
+	defaultName          = "IBMProviderClient"
```

Comment:
I think we should give some relevant variable name. `defaultName` is not clearly indicating which default name it is representing.

Reply:
It is representing default name for IBMProviderStorage server

Reply:
changd it to `IBMGrpcProvider`